testsuite/testsuite-buildtool/multi-module/view/src/test/java/VerifyBuildArtefactIT.java

```diff
@@ -29,32 +26,9 @@ public static Archive createDeployment() throws Exception {
 
     @org.junit.Test
     public void testEndppoint() throws Exception {
-        String urlContents = getUrlContents("http://127.0.0.1:8080/");
+        String urlContents = IOUtils.getUrlContents("http://127.0.0.1:8080/");
         Assert.assertEquals("something\n", urlContents);
     }
 
-    private static String getUrlContents(String theUrl) {
```

Comment:
This method body was duplicated in multiple test classes. Moved the code in to a separate utility file under the Arquillian module and replace all occurrence that were identified by the IDE>